### PR TITLE
Require PHP 7.4 and Symfony 5.4

### PR DESCRIPTION
**Pull request by mbabker**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Doc updated   | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT

Similar to https://github.com/schmittjoh/JMSSerializerBundle/pull/937, this proposes bumping the minimum library requirements to PHP 7.4 and Symfony 5.4.  Older Symfony versions are no longer supported, and the [Packagist stats](https://packagist.org/packages/jms/serializer/php-stats#3) show only ~3% of recent 3.x downloads are on PHP 7.2 and 7.3.

### CODE REVIEW

File: .github/workflows/benchmark.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - "7.2"
+          - "7.4"
           - "8.0"
           - "8.1"
```
Comment (scyzoryck) [inline .github/workflows/benchmark.yaml:19]:
```suggestion
          - "7.4"
          - "8.2"
```

**Review by scyzoryck — APPROVED** [review]
Thanks! It sounds reasonable :) 
I left one small comment and I think we can merge it into Mondays release! 🎉 

Best, scyzoryck

**Comment by scyzoryck** [comment]
Looks good for me! @goetas - do you think it should be released under 3.x or under new major version?

**Comment by goetas** [comment]
increasing the min required version it is not a BC break, it can be done in a feature release, no major required

**Pull request merged**